backend/api/management/commands/generate_fixtures.py

```diff
@@ -1,218 +1,297 @@
+        ]:
+            self.create_basic_cp(status)
+        self.create_aggregate_cp()
+        self.create_composite_cp()
+
+    def create_algos(self):
+        logger.debug("  Create algos")
+        self.simple_algo = factory.create_algo(
             inputs=factory.build_algo_inputs(["datasamples", "opener", "model"]),
             outputs=factory.build_algo_outputs(["model"]),
-            name="simple algo",
+            name="simple",
         )
-        aggregate_algo = factory.create_algo(
-            inputs=factory.build_algo_inputs(["model"]),
+        factory.create_algo_files(
+            key=self.simple_algo.key,
+            description=ContentFile("Simple algo"),
+        )
+
+        self.aggregate_algo = factory.create_algo(
+            inputs=factory.build_algo_inputs(["models"]),
             outputs=factory.build_algo_outputs(["model"]),
             name="aggregate",
         )
-        composite_algo = factory.create_algo(
+        factory.create_algo_files(
+            key=self.aggregate_algo.key,
+            description=ContentFile("Aggregate algo"),
+        )
+
+        self.composite_algo = factory.create_algo(
             inputs=factory.build_algo_inputs(["datasamples", "opener", "local", "shared"]),
             outputs=factory.build_algo_outputs(["local", "shared"]),
             name="composite",
         )
-        metric_algo = factory.create_algo(
+        factory.create_algo_files(
+            key=self.composite_algo.key,
+            description=ContentFile("Composite algo"),
+        )
+
+        self.predict_algo = factory.create_algo(
+            inputs=factory.build_algo_inputs(["datasamples", "opener", "model", "shared"]),
+            outputs=factory.build_algo_outputs(["predictions"]),
+            name="predict",
+        )
+        factory.create_algo_files(
+            key=self.predict_algo.key,
+            description=ContentFile("Predict algo"),
+        )
+
+        self.metric_algo = factory.create_algo(
             inputs=factory.build_algo_inputs(["datasamples", "opener", "predictions"]),
             outputs=factory.build_algo_outputs(["performance"]),
             name="metric",
         )
+        factory.create_algo_files(
+            key=self.metric_algo.key,
+            description=ContentFile("Metric algo"),
+        )
+
+    def create_data_manager(self):
+        logger.debug("  Create data manager")
+        self.data_manager = factory.create_datamanager()
+        factory.create_datamanager_files(key=self.data_manager.key)
+
+    def create_data_samples(self):
+        logger.debug("  Create data samples")
+        self.train_data_sample_keys = []
+        self.predict_data_sample_keys = []
+        self.test_data_sample_keys = []
 
-        data_manager = factory.create_datamanager()
-        train_data_sample = factory.create_datasample([data_manager])
-        test_data_sample = factory.create_datasample([data_manager], test_only=True)
+        for _ in range(3):
```

Comment:
could you add a comment as done in line 118. why we are creating 3 set of train datasamples ?

Reply:
This is not a requirement. 3 is an arbitrary number.

Reply:
I add a comment to explain this, tell me if it's not clear